src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: Print API SQS Message Types
-  version: '1.4.0'
+  version: '1.5.0'
   description: |-
     Print API SQS Message Types
```

Comment:
Can you put a dummy `path` and `post` method in here please (like we do with the others)
I know its not a REST API !! (you're telling the wrong man 😁 ) , but we're using the `path` elements to "document" [sic] the queue which then hooks up to the model as its request payload.
Have a look at what we've done with the others

Reply:
Ofcourse 😆 . Code pushed 👍🏼